front/app/containers/Admin/pagesAndMenu/routes.tsx

```diff
@@ -61,7 +60,7 @@ export default () => ({
           index: true,
           element: (
             <PageLoading>
-              <NavigationSettings />
+              <Outlet id="app.containers.Admin.pages-menu.NavigationSettings" />
             </PageLoading>
           ),
```

Comment:
@brentguf 

> Btw, is the linkTo in this file correct? ('/admin/pages-menu/pages/new')

Yes, I believe so https://github.com/CitizenLabDotCo/citizenlab/blob/fa6869d44c2a2b50564055e629bfe01b51085d8e/front/app/containers/Admin/pagesAndMenu/routes.tsx#L102-L106

> Wdyt?

For me, it looks like again we're trying to do not really necessary refactoring of the code that was neither just created nor changed. It will again increase the complexity of this PR, take more time postponing the release again, and possibly introduce new bugs. So, I would rather avoid it. But of course, we can do it later in a new PR.

Reply:
Agree, would just make the code a bit easier to understand. But I get you're busy with other stuff.